llvm/lib/Transforms/Utils/SCCPSolver.cpp

```diff
@@ -873,6 +890,15 @@ bool SCCPInstVisitor::markConstant(ValueLatticeElement &IV, Value *V,
   return true;
 }
 
+bool SCCPInstVisitor::markConstantRange(ValueLatticeElement &IV, Value *V,
+                                        ConstantRange &CR) {
```

Comment:
Should pass by const reference here (or by value and use std::move, but that's not compatible with the dbg printing).

Reply:
added const